lib/HTML/Form.pm

```diff
@@ -506,7 +506,11 @@ input with the given name and/or type.
 sub find_input
 {
     my($self, $name, $type, $no) = @_;
+    die "Invalid index $no.\n"
```

Comment:
All the existing calls to `die` do not have a newline at the end, so will include our line number. Is there a reason you've chosen to change that behaviour here?

Reply:
It was an oversight. I'll create a fixup.